docs/user-guide/deployments-administration/deploy-on-kubernetes/configure-frontend-groups.md

```diff
@@ -31,57 +31,49 @@ The configuration structure has changed between chart versions:
 Always refer to the latest [values.yaml](https://github.com/GreptimeTeam/helm-charts/blob/main/charts/greptimedb-cluster/values.yaml) in the Helm chart repository for the most up-to-date configuration structure.
 :::
 
-When defining the frontend groups, you must specify the name field for each frontend instance. Below is an example configuration that creates read and write frontend replicas:
+When configuring frontend groups, ensure that each group includes a `name` field. The following `values.yaml` example demonstrates how to define separate frontend groups for read and write operations:
 
 ```yaml
-apiVersion: greptime.io/v1alpha1
-kind: GreptimeDBCluster
-metadata:
-  name: greptimedb
-  namespace: default
-spec:
-  initializer:
-    image: greptime/greptimedb-initializer:latest
-  base:
-    main:
-      image: greptime/greptimedb:latest  
-  frontendGroups:
-  - name: read
-    replicas: 2
-    config: |
-      default_timezone = "UTC"
-      [http]
-      timeout = "60s"
-  - name: write
-    replicas: 1
-    config: |
-      default_timezone = "UTC"
-      [http]
-      timeout = "60s"
-  meta:
-    replicas: 1
-    backendStorage:
-      etcd:
-        endpoints:
-          - "etcd.etcd-cluster.svc.cluster.local:2379"
-  datanode:
-    replicas: 1
+frontendGroups: 
```

Comment:
Better to disable frontend: https://github.com/GreptimeTeam/helm-charts/blob/main/charts/greptimedb-cluster/values.yaml#L139
```
frontend:
  enabled: false
```

Reply:
[67ae98e](https://github.com/GreptimeTeam/docs/pull/2125/commits/67ae98ecc9a774ff7d82c4629a249c3bcdb061fc)